### PR TITLE
Add infrastructure tags to Enos scenarios

### PR DESCRIPTION
**Pull request by jaymalasinha**
Signed-off-by: Jaymala Sinha <jaymala@hashicorp.com>

### CODE REVIEW

File: enos/enos-variables.hcl [hunk 1]
```diff
@@ -52,6 +52,18 @@ variable "alb_listener_api_port" {
...
+variable "project_name" {
+  description = "The description of the project"
+  type        = string
+  default     = "boundary-enos-integration"
+}
+
+variable "tags" {
+  description = "Tags that will be applied to infrastructure resources that support tagging"
+  type        = map(string)
+  default     = null
```
Comment (mikebaum) [inline enos/enos-variables.hcl:64]: I wonder if this default value should rather be: `{}`. Not sure if merge function works well with a `null` argument for any of its args. Maybe you could just double check that it works with null.

**Review by mikebaum — COMMENTED** [review]
Just one question about the default value for the tags variable, otherwise everything looks good.

File: enos/enos-variables.hcl (hunk 1 above)
Comment (jaymalasinha) [reply enos/enos-variables.hcl:64]: it does seem to work with default `null`, as we have the same setup in Vault scenarios.

**Review by mikebaum — APPROVED** [review]
Looks good to me.

**Review by joshbrand — APPROVED** [review]

**Pull request merged**